app/models/placement.rb

```diff
@@ -63,4 +64,8 @@ def assign_subject
 
     update!(subject: subjects.last)
   end
+
+  def additional_subject_names
+    additional_subjects.map(&:name).sort
+  end
```

Comment:
I'm not entirely convinced this needs to be on the model. It feels more appropriate for the decorator since it's primarily a view concern (and in fact, that's the only place this method is used currently.)